guides/common/modules/proc_installing-flatpak-applications.adoc

```diff
@@ -0,0 +1,40 @@
+[id="installing-flatpak-applications_{context}"]
+= Installing Flatpak applications
+
+This procedure installs selected applications from the enabled Flatpak remotes on the command line.
+
+.Prerequisites
+* Flatpak is installed.
+* The Red Hat Flatpak remote is enabled.
+
+.Procedure
+. Run the following command on the managed host to add the {Project} as a Flatpak remote:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak --user remote-add --authenticator-name=org.flatpak.Authenticator.Oci katello oci+https://satellite.example.com/
+----
+. Log in using podman:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ podman login
+----
+. Install the Mozilla Firefox Flatpak:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+$ flatpak install firefox
+----
+
+.Procedure using remote execution templates
+
+. Set up Flatpak remote on host:
+* A user-provided name for the {Project} or {SmartProxy} remote.
```

Comment:
One more required param here for the 1st REX template is :
Flatpak registry URL: Satellite_URL for satellite, capsule_url/pulpcore_registry/ for clients getting content from capsules.